libs/libarchfpga/src/read_fpga_interchange_arch.cpp

```diff
@@ -34,6 +34,72 @@ using namespace DeviceResources;
 using namespace LogicalNetlist;
 using namespace capnp;
 
+static float get_corner_value(Device::CornerModel::Reader model, const char* speed_model, const char* value) {
```

Comment:
The corner values are something new. Should we add sufficient comments about it models the ``speedgrade`` and ``process corners``? Also should we comment the strategy to find the timing value under a given speed grade and a corner?

Reply:
Added some more comments around this, also redirecting to the schema where there is a description of the timing model structure.